in-payments.js

```diff
@@ -0,0 +1,25 @@
+const paymentsIn = [
+    {
+      date: "2017-10-17",
+      currency: "GBP",
+      amount: 40.06,
+      exchangeRate: 1,
+      description: "New bike",
+    },
+    {
+      date: "2017-10-17",
+      currency: "GBP",
+      amount: 2.10,
+      exchangeRate: 1,
+      description: "Coffee with Luke at iCafe",

```

Comment:
Incoming Payment can't have description as `Coffee with Luke at iCafe`
this can be `Salary for October` :)